srtcore/core.cpp

```diff
@@ -7786,10 +7745,33 @@ void srt::CUDT::sendCtrl(UDTMessageType pkttype, const int32_t* lparam, void* rp
         m_tsLastSndTime.store(steady_clock::now());
 }
 
+// [[using locked(m_RcvBufferLock)]]
+bool srt::CUDT::getFirstNoncontSequence(int32_t& w_seq, string& w_log_reason)
```

Comment:
This is consistent with the future implementation of it where it returns false in case when it neither has a receiver buffer nor is a group member. This situation should never happen and therefore for this it's a sanity check. I can change it, but this will cause merge problems in the future.